spark-atlas-connector/src/test/scala/com/hortonworks/spark/atlas/sql/SparkExecutionPlanProcessorForStreamingQuerySuite.scala

```diff
@@ -183,6 +183,72 @@ class SparkExecutionPlanProcessorForStreamingQuerySuite
     }
   }
 
+  test("SAC-253 Kafka source(s) to kafka sink - micro-batch query" +
```

Comment:
This UT is basically running same query with above UT, but activating 1 topic per batch via sending message to topic later.